wordvecspace/command.py

```diff
@@ -1,12 +1,20 @@
 import code
 
 from basescript import BaseScript
-from convert import GWVec2WordVecSpace
-from wordvecspace import WordVecSpace
+from .convert import GWVec2WordVecSpace
+from .wordvecspace import WordVecSpace
+from .tornado_wvs import *
 
 class WordVecSpaceCommand(BaseScript):
     DESC = 'Word Vector Space command-line tool'
 
+    DEFAULT_PORT = 8900
+
+    def runserver(self):
+        app = make_app()
+        app.listen(self.args.port)
+        tornado.ioloop.IOLoop.current().start()
```

Comment:
@RamanjaneyuluIdavalapati This tornado logic should be inside "service.py". Something like

```python
start_service(self.args.port)
```

should suffice or something like this.

```python
service = Service(self.args.port)
service.start()
```

Reply:
@prashanthellina done